src/server/req_manager.c

```diff
@@ -1269,10 +1269,9 @@ mgr_queue_delete(struct batch_request *preq)
 		}
 
 		if (rc == 0) {
-			char queue_name[PBS_MAXQUEUENAME];
+			char queue_name[PBS_MAXQUEUENAME + 1];
 			/* Save the queue name before we purge it so it will appear in the log.  */
 			strncpy(queue_name, pque->qu_qs.qu_name, sizeof(queue_name));
-			queue_name[(sizeof(queue_name) - 1)] = '\0';
```

Comment:
why remove this line?

Reply:
strncpy() copy complete src data including null.

Reply:
The strncpy() function is similar, except that at most n bytes of src are copied. Warning: If there is no null byte among the first n bytes of src, the string placed in dest will not be null-terminated.

reference:https://linux.die.net/man/3/strncpy

Reply:
@prakashcv13 is right, if strncpy() will leave the string unterminated if it truncates.  We have our own version of this to handle this.  Use pbs_strncpy().